tests/cli-app/test/project.js

```diff
@@ -24,12 +24,12 @@ function registerProjectHooks (network) {
     }
   })
   
-  after('cleaning up project folder', function () {
-    run('rm build/contracts/*.json ||:')
-    run('rm contracts/*.sol ||:')
-    run('rm zos.* ||:')
-    run('rm package.* ||:')
-  })
+  // after('cleaning up project folder', function () {
+  //   run('rm build/contracts/*.json ||:')
+  //   run('rm contracts/*.sol ||:')
+  //   run('rm zos.* ||:')
+  //   run('rm package.* ||:')
+  // })
```

Comment:
should we remove this `after` statement?

Reply:
No, I unfortunately forgot to uncomment it. It's no harm though.